src/tmpfiles/tmpfiles.c

```diff
@@ -1126,46 +1127,104 @@ static int parse_acls_from_arg(Item *item) {
 }
 
 #if HAVE_ACL
+static int set_acl_cond_exec(const char *path, acl_type_t type, acl_t *acl) {
+        _cleanup_(acl_freep) acl_t old;
```

Comment:
Please initialize `old` with `NULL`, otherwise, the cleanup function may use uninitialized value.